df3d/core.py

```
@@ -311,11 +312,15 @@ def plot_2d(self, cam_id, img_id, with_corrections=False, smooth=False, joints=[
         from pyba.config import df3d_bones, df3d_colors
 
         if with_corrections:
-            pts2d = self.corrected_points2d(cam_id, img_id)
+            pts = self.corrected_points2d(cam_id, img_id)
         else:
-            pts2d = None
+            pts = None
+
+        if reprojection:
+            pts = np.copy(self.camNet.points3d)
```

Should we raise an error if the user asks for `with_corrections=True` and `reprojection=True`? I don't understand exactly what happens in `self.corrected_points2d`, but I would guess these two arguments are incompatible, is that true? If so, adding `if with_corrections and reprojection: raise ValueError('"with_corrections" and "reprojection" cannot both be set to True')` near the start of this function would be great.